client/state/themes/test/actions.js

```diff
@@ -213,8 +208,8 @@ describe( 'actions', () => {
 					} );
 			} );
 
-			test( 'should dispatch fetch action when thunk triggered', () => {
-				requestThemes( 'wpcom' )( spy );
+			test( 'should dispatch fetch action when thunk triggered', async () => {
+				await requestThemes( 'wpcom' )( spy );
```

Comment:
Necessary because of the recent changes in how the scope is being marked as done (see [point 6 here](https://github.com/nock/nock/blob/75507727cf09a0b7bf0aa7ebdf3621952921b82e/migration_guides/migrating_to_13.md)).